Assets/Scripts/BulletTrigger.cs

```diff
@@ -6,8 +6,8 @@ public class BulletTrigger : MonoBehaviour {
 
     void OnCollisionEnter(Collision collision) {
         if (collision.gameObject.tag == "Player") {
-            PlayerController controller = collision.gameObject.GetComponent("PlayerController") as PlayerController;
-            controller.KillPlayer();
+            PlayerLife playerLife = collision.gameObject.GetComponent("PlayerLife") as PlayerLife;
```

Comment:
`PlayerLife playerLife = collision.gameObject.GetComponent<PlayerLife>();`
Is there any reason youve been getting components like this? Wondering if you know something I dont

Reply:
The examples I've been doing stuff off of have done it this way, only reason.